airflow/www/extensions/init_jinja_globals.py

```diff
@@ -69,10 +70,17 @@ def prepare_jinja_globals():
             "git_version": git_version,
             "k8s_or_k8scelery_executor": IS_K8S_OR_K8SCELERY_EXECUTOR,
             "rest_api_enabled": False,
-            "auth_manager": get_auth_manager(),
             "config_test_connection": conf.get("core", "test_connection", fallback="Disabled"),
         }
 
+        # Extra global specific to auth manager
+        extra_globals.update(
+            {
+                "auth_manager": get_auth_manager(),
+                "DagDetails": DagDetails,
+            }
+        )
```

Comment:
```suggestion
        extra_globals["auth_manager"] = get_auth_manager()
        extra_globals["DagDetails"] = DagDetails
```

Feels easier to read and actually shorter.

Reply:
Agree, this is better